fluentlenium-core/src/test/java/org/fluentlenium/core/action/WindowActionsTest.java

```diff
@@ -0,0 +1,280 @@
+package org.fluentlenium.core.action;
+
+import com.google.common.collect.ImmutableSet;
+import org.assertj.core.api.Assertions;
+import org.fluentlenium.configuration.ConfigurationProperties;
+import org.fluentlenium.core.FluentDriver;
+import org.fluentlenium.core.components.ComponentsManager;
+import org.fluentlenium.core.domain.FluentWebElement;
+import org.fluentlenium.core.wait.FluentWait;
+import org.fluentlenium.core.wait.FluentWaitWindowMatcher;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebDriver;
+
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
I should use this too, instead of invoking MockitoAnnotations.init manually :)

:+1: